### PR TITLE
Stubs for most Bitmapdata methods for AVM2 and AVM1

### PR DESCRIPTION
**Pull request by tschneidereit**
r? @yurydelendik, mostly for the AVM1 wrappers.

### CODE REVIEW

File: src/avm1/lib/AVM1BitmapData.ts [hunk 1]
```diff
@@ -113,10 +114,18 @@ module Shumway.AVM1.Lib {
     }
 
     public copyChannel(sourceBitmap: AVM1BitmapData, sourceRect: AVM1Object, destPoint: AVM1Object, sourceChannel: number, destChannel: number): void {
-      Debug.notImplemented('AVM1BitmapData.copyChannel');
+      var as3BitmapData = sourceBitmap.as3BitmapData;
+      var as3SourceRect = toAS3Rectangle(sourceRect);
+      var as3DestPoint = toAS3Point(destPoint);
+      sourceChannel = alCoerceNumber(this.context, sourceChannel);
+      destChannel = alCoerceNumber(this.context, destChannel);
+      return this.as3BitmapData.copyChannel(as3BitmapData, as3SourceRect, as3DestPoint,
```
Comment (yurydelendik) [inline src/avm1/lib/AVM1BitmapData.ts]: Remove 'return'

**Comment by yurydelendik** [comment]
r+ looks good

**Comment by tschneidereit** [comment]
Thanks for the quick review!

**Pull request merged**